apps/sms/views/new_message/style/recipients.css

```diff
@@ -97,7 +97,7 @@ Do not remove.
   width: auto;
 }
 
-.recipient[contenteditable].email {
+.recipient[contenteditable=false].email {
```

Comment:
I think the issue we have here is that we have a specific style for email pills with padding and the cascade and specificity stuff makes this awkward.

I don't think the .email part is in the BB.

So all this means we should do that in a separate bug :)